frontend/src/components/TriviaTrueFalse.tsx

```diff
@@ -0,0 +1,64 @@
+import {useEffect, useState} from "react";
+import {TriviaQuizData} from "../model";
+
+export default function TriviaTrueFalse() {
+
+    const [amount, setAmount] = useState(1 as number)
```

Comment:
```suggestion
    const [amount, setAmount] = useState(1)
```

Is as number necessary? not sure about this one, but should work without, since your default value is a number